src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2487,6 +2487,25 @@ private static bool IsUnsignedIntegralType(TypeSymbol type)
             }
         }
 
+        internal static void GetEffectiveParameterTypes(
```

Comment:
>GetEffectiveParameterTypes [](start = 29, length = 26)

Is this method used?

Reply:
The method is called in `NullableWalker.InferMethod`.